src/lib/features/feature-toggle/feature-toggle-service.ts

```diff
@@ -2444,7 +2450,12 @@ class FeatureToggleService {
     }
 
     async setFeatureCreatedByUserIdFromEvents(): Promise<void> {
-        await this.featureToggleStore.setCreatedByUserId(100);
+        const updated = await this.featureToggleStore.setCreatedByUserId(100);
+        if (updated > -1) {
+            this.eventBus.emit(FEATURES_CREATED_BY_PROCESSED, {
+                updated,
+            });
+        }
```

Comment:
I think it makes sense if we say that zero were updated to do this:
```suggestion
        if (updated > 0) {
            this.eventBus.emit(FEATURES_CREATED_BY_PROCESSED, {
                updated,
            });
        }
```

Reply:
I'll change this to check for undefined